src/third_party/asio-asio-1-11-0/asio/include/asio/impl/error_code.ipp 
--- src/third_party/asio-asio-1-11-0/asio/include/asio/impl/error_code.ipp.orig	2016-01-12 14:04:28.770860071 +0100
+++ src/third_party/asio-asio-1-11-0/asio/include/asio/impl/error_code.ipp	2016-01-12 14:05:21.615858764 +0100
@@ -97,10 +97,7 @@ public:
 #if defined(__sun) || defined(__QNX__) || defined(__SYMBIAN32__)
     using namespace std;
     return strerror(value);
-#elif defined(__MACH__) && defined(__APPLE__) \
-  || defined(__NetBSD__) || defined(__FreeBSD__) || defined(__OpenBSD__) \
-  || defined(_AIX) || defined(__hpux) || defined(__osf__) \
-  || defined(__ANDROID__)
+#elif ! defined(__GLIBC__)
     char buf[256] = "";
     using namespace std;
     strerror_r(value, buf, sizeof(buf));
